IncludeToolbox/Options/FormatterOptionsPage.cs

```diff
@@ -19,16 +19,29 @@ public enum PathMode
             Shortest,
             Shortest_AvoidUpSteps,
             Absolute,
+            Absolute_FromParentDirWithFile
```

Comment:
This not actually absolute in any sense. It is relative from where the special file is placed, so I'd call it `ForceRelativeToParentDirWithFile` since it will always try to make the path relative to the special folder, blindly assuming that this will compile. In contrast all the other modes ensure that the the path can be resolved with the IncludePaths in the project file or by being relative to the including source file.

Please correct me if I got this wrong :)

Reply:
No, you're right. I think your name will make more sense. In our workflow I just tend to think about them as "absolute with respect to the code base" vs. "relative to the current file".